datasets/smart_contract_audit_findings/README.md

```diff
@@ -0,0 +1,10 @@
+# Smart Contract Audit Findings
+
+Format of findings files is CSV containing:
```

Comment:
Maybe: "Audit findings are presented in a CSV with the following fields:"

Reply:
Done